environments/environment-DEV-OSX.yml

```diff
@@ -1,4 +1,5 @@
-name: watershed_workflow_DEV-2021-11-10
+#name: watershed_workflow_DEV-2021-11-10
+name: watershed_workflow_Dev
```

Comment:
don't change environments files in the repo -- I'm not sure why they are getting changed by everyone.  Must be in the installation process?  @ecoon should look into this at some point